### ISSUE TITLE
gen-manifest breaks when cli values contain spaces

### ISSUE THREAD

**Issue by vchudnov-g** [googleapis/sample-tester#106]
In gen-manifest, user-supplied key values that contain spaces do not emit correct manifests.

In the sample tester repo, compare the output of the following lines:

```
# works
gen-manifest --bin=test-bin --env=java --chdir=samples --output=/tmp/2019-08-05.samples.manifest.bin.yaml tests/testdata/gen_manifest/*

# does not work
gen-manifest --bin="test-bin foo" --env=java --chdir=samples --output=/tmp/2019-08-05.samples.manifest.bin.space.foo.yaml tests/testdata/gen_manifest/*

# works
gen-manifest --invocation="custom-command" --env=java --chdir=samples --output=/tmp/2019-08-05.samples.manifest.invocation.yaml tests/testdata/gen_manifest/*

# does not work
gen-manifest --invocation="custom-command @args foo" --env=java --chdir=samples --output=/tmp/2019-08-05.samples.manifest.invocation.space.yaml tests/testdata/gen_manifest/*
```

In the failing cases, the manifest looks something like this:
```
type: manifest/samples
schema_version: 3
base: &common
  env: 'java'
  chdir: 'samples'
  sts/testdata/gen_manifest/getbook.py: ''
  sts/testdata/gen_manifest/readbook.py: ''
  basepath: '.'
samples:
```

I suspect a problem in the arg parsing.....

**Comment by vchudnov-g** [comment]
The underlying issue appears to be a Python bug in argparsing when the arg value contains a space. [This bug report](https://bugs.python.org/issue22909) is seemingly the issue, though it dates from 2014 and looks like it should have been resolved. I will see whether Python 3.7's `parse_intermixed_arguments()` addresses this.

**Comment by vchudnov-g** [comment]
Fixed by #108
